packages/perseus-editor/src/__tests__/item-extras-editor.test.tsx

```diff
@@ -58,15 +58,15 @@ describe("ItemExtrasEditor", () => {
                 onChange={onChangeMock}
             />,
         );
-        const checkbox = screen.getByLabelText("Show periodic table:");
+        const checkbox = screen.getByLabelText("Show periodic table");
 
         // Act
         await userEvent.click(checkbox);
 
         // Assert
         // visible when periodicTable is checked
         expect(
-            screen.getByText("Include key/legend with periodic table:"),
+            screen.getByText("Include key/legend with periodic table"),
```

Comment:
Nit - optional: I think it's supposed to be better practice to use `getByRole` if possible? So I'd probably do `screen.getByRole("checkbox", {name: "Include key/legend with periodic table"})`